ios/FluentUI/Extensions/UIScreen+Extension.swift

```diff
@@ -1,29 +0,0 @@
-//
-//  Copyright (c) Microsoft Corporation. All rights reserved.
-//  Licensed under the MIT License.
-//
-
-import UIKit
-
-extension UIScreen {
-    var devicePixel: CGFloat { return 1 / scale }
-
-    internal func roundToDevicePixels(_ value: CGFloat) -> CGFloat {
-        /*
-         Round to 3 digits after floating point to better match the device rounding on 3x devices.
-         Avoids a situation where calculated size is smaller than screen size by a difference in 4th+ position after floaing point.
-         For example:
-         calculated = 52.666666666666671
-         actual = 52.666667938232422
```

Comment:
Is this still an issue? Do we have an actual bug of this we fixed we can check against?

Reply:
i assume this bug still exists. however, i think we should avoid aliasing as much as we can in our frame logic